minefana-bungeecord/src/main/java/com/github/games647/minefana/BungeeListener.java

```diff
@@ -17,12 +20,19 @@
 
     @EventHandler(priority = EventPriority.HIGHEST)
     public void onPlayerJoin(ServerConnectedEvent joinEvent) {
-        plugin.getPlayerCollector().onPlayerJoin(joinEvent.getPlayer());
+        plugin.getProxy().getScheduler().schedule(plugin, new Runnable() {
```

Comment:
Could you do that for the other platforms too?

Reply:
ok